WcaOnRails/config/locales/en.yml

```diff
@@ -925,11 +936,23 @@ en:
         one: "WCA Delegate"
         other: "WCA Delegates"
       contact: "Contact"
-      competitor_limit: "Competitor Limit"
-      entry_fee: "Entry Fee"
       information: "Information"
       events: "Events"
       registration_requirements: "Registration requirements"
+      create_wca_account_html: "Create a WCA account %{here} if you don't have one."
+      claim_wca_id_html: "If this is not your first competition associate your WCA ID to your WCA account %{here}."
```

Comment:
nit: I think we should add a comma between "competition" and "associate"

Reply:
Ok!